### PR TITLE
beta server header styles

### PR DESCRIPTION
**Pull request by SBBCornell**
This PR adds indicators to let users know they are on beta, not the main site. Its currently running on https://beta.arxiv.org/

The indicators are:
- header background color
- link to main site
- exclamation icon

I would also like to test it on dev, could I get help with that?

screenshot:
Screen Shot 2020-07-27 at 3 30 53 PM

### CODE REVIEW

**Comment by mhl10** [comment]
@SBBCornell reports that reviewers should look at lines that check what server is being accessed.

File: browse/templates/abs/abs.html [hunk 1]
```diff
@@ -18,7 +18,11 @@
...
+  <div class="header-breadcrumbs is-hidden-mobile">
+    {% if config['BROWSE_SITE_HOST'] != None %}<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="beta-exclamation"><path d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"/></svg>{% endif %}
+    <a href="/">{{ config['BROWSE_SITE_LABEL'] }}</a> &gt; <a href="{{ url_for('.list_articles',context=abs_meta.primary_archive.id, subcontext='recent') }}">{{ abs_meta.primary_archive.id }}</a> &gt; arXiv:{{ requested_id }}
+    {% if config['BROWSE_SITE_HOST'] != None %}<a href="https://arxiv.org/" class="return-link-abs">(go to arxiv.org)</a>{% endif %}
```
Comment (mhl10) [inline browse/templates/abs/abs.html:31]: These are the lines in question to review.

Comment (DavidLFielding) [reply browse/templates/abs/abs.html:31]: Will it always be the case that setting BROWSE_SITE_HOST will indicate the host is a development box? Would this be set in a similar way for mirrors (once we update mirrors to use new browse)? May not be a factor now.

Comment (mhl10) [reply browse/templates/abs/abs.html]: I've made a small change here to check for "dev" or "beta" in `BROWSE_SITE_LABEL`.

@SBBCornell if you'd like to test this locally you can run:

```
BROWSE_SITE_LABEL='dev.arxiv.org' FLASK_APP=app.py FLASK_DEBUG=1 pipenv run flask run
```

I tried messing with the viewbox settings but couldn't get the exclamation icon smaller:
Screen Shot 2020-11-08 at 9 58 17 PM

**Comment by SBBCornell** [comment]
Here are new screenshots of the homepage and abs page:

Screen Shot 2020-11-11 at 12 20 47 PM

Screen Shot 2020-11-11 at 12 21 02 PM

**Review by DavidLFielding — APPROVED** [review]
Looks good. Not sure the css classes need to be named beta-*. That doesn't impact the indicator functionality which is urgently needed.

**Review by mhl10 — APPROVED** [review]

**Pull request closed without merging**